specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -12546,6 +12552,18 @@
             "modelAsString": true
           }
         },
+        "assessmentMode": {
```

Comment:
When adding a new optional property, please put it at the end of the list of properties.